From 842f9ce85bb422252a301aac6f86bde82329a95e Mon Sep 17 00:00:00 2001
From: Salvatore Bonaccorso <carnil@debian.org>
Date: Thu, 20 Sep 2012 15:36:58 +0200
Subject: [PATCH 1/2] Fix typo to path to shlibs files

Fix typo in "8.6.4.1 The shlibs files present on the system". Refer to
/var/lib/dpkg/info/*.shlibs for the location of the shlibs files.
---
 policy.sgml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy.sgml b/policy.sgml
index 5760a3f..ba217e5 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -6694,7 +6694,7 @@ Built-Using: grub2 (= 1.99-9), loadlin (= 1.6e-1)
 		    The <file>shlibs</file> control files for all the
 		    packages currently installed on the system.  These are
 		    normally found
-		    in <file>/var/lib/dpkg/info/*.symbols</file>, but
+		    in <file>/var/lib/dpkg/info/*.shlibs</file>, but
 		    packages should not rely on this and instead should
 		    use <tt>dpkg-query --control-path <var>package</var>
 		    shlibs</tt> if for some reason these files need to be
-- 
1.7.10.4

